tools/data_builder.py

```diff
@@ -122,6 +127,27 @@ def find_md_link_or_url(text):
     return text_content.strip(), link_url.strip()
 
 
+def _format_lat_or_long(val: str) -> None:
+    return val.strip("+")
+
+
+def validate_geo(geo_body_raw: str) -> str:
+    geo_body = geo_body_raw.strip()
+    if geo_body == "":
+        return ""
+
+    result = GEOLOCATION_REGEX.match(geo_body)
+
+    try:
+        result_1 = _format_lat_or_long(result.group(1))
+        result_2 = _format_lat_or_long(result.group(2))
+    except AttributeError:
+        ValueError(f"Could not parse geolocation: {geo_body}")
+    if result:
+        return f"{result_1}, {result_2}"
```

Comment:
Find to skip splitting up regex, not the biggest issue esp. with new comment. Tiny suggestiong to change var names for more self-documenting code:

```suggestion
        lat = _format_lat_or_long(result.group(1))
        long = _format_lat_or_long(result.group(2))
    except AttributeError:
        ValueError(f"Could not parse geolocation: {geo_body}")
    if result:
        return f"{lat}, {long}"
```